target/linux/realtek/image/Makefile

```diff
@@ -90,12 +90,12 @@ define Device/Default
   KERNEL := \
 	kernel-bin | \
 	append-dtb | \
-	gzip | \
+	libdeflate | \
```

Comment:
I suspect people are more likely to understand what a `gzip` command does, compared to `libdeflate`.  Not knowing libdeflate, I was wondering if it only supports gzip, or if it also supports other compression methods. Looking at their README, this looks exactly like the "DEFLATE vs. zlib vs. gzip" ambiguity that is addressed there.

Should this be changed to something more explicit? Maybe `libdeflate-gzip` like the actual command?

Reply:
Lets do that. While I encourage people learning new things, making things easier to read and understand surely is only beneficial :)

Reply:
This is a good idea to make it clear it's compatible and not breaking change.